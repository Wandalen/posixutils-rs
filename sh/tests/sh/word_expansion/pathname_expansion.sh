cd tests/read_dir

echo dir/inner_dir/data1
echo dir/inner_dir/*
echo dir?inner_dir?data1
echo dir[x/y]inner_dir[x/y]data1
echo dir[[:graph:]]inner_dir[[:graph:]]data1
echo */*/*

echo *
echo ?hidden
echo *hidden
echo [!a]hidden
echo [[:punct:]]hidden
echo .*

echo file?.txt
echo f*.*
echo file[[:digit:]].txt
