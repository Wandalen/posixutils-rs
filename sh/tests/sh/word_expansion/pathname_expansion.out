dir/inner_dir/data1
dir/inner_dir/data1 dir/inner_dir/data2 dir/inner_dir/data3
dir?inner_dir?data1
dir[x/y]inner_dir[x/y]data1
dir[[:graph:]]inner_dir[[:graph:]]data1
dir/inner_dir/data1 dir/inner_dir/data2 dir/inner_dir/data3
dir file1.txt file2.txt filea.txt filea.xyz fileb.txt filec.txt
?hidden
*hidden
[!a]hidden
[[:punct:]]hidden
. .. .hidden
file1.txt file2.txt filea.txt fileb.txt filec.txt
file1.txt file2.txt filea.txt filea.xyz fileb.txt filec.txt
file1.txt file2.txt
